core/jvm/src/test/scala/scalaz/zio/QueueSpec.scala

```diff
@@ -120,7 +120,7 @@ class QueueSpec(implicit ee: ExecutionEnv) extends Specification with AroundTime
   def e10 = unsafeRun(
     for {
       queue <- Queue.bounded[Nothing, Int](0)
-      f     <- queue.offer[Nothing](1).fork[Nothing, Unit]
+      f     <- queue.offer(1).fork[Nothing, Unit]
```

Comment:
`fork` and `fork0` no longer need the separate type parameter; because `fork` / `fork0` cannot fail, they can simply return `Nothing` for their type parameters.

```scala
def fork: IO[Nothing, Fiber[E, A]]
```